.changeset/brown-poets-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/nextjs': patch
+---
+
+Exports SignInWithMetamaskButton for @clerk/nextjs package
```

Comment:
This will be added to a changelog, so it should be phrased similar to the below:
```suggestion
`SignInWithMetamaskButton` is now exported from `@clerk/nextjs` package
```